src/components/utils/skills-quiz.jsx

```diff
@@ -0,0 +1,7 @@
+import { DROPDOWN_OPTION_IMPROVE_CURRENT_ROLE } from '../skills-quiz/constants';
```

Comment:
[nit/curious/non-blocking] curious why this file needed to be renamed, given no JSX exists in the file?

Reply:
Ahh this refactored when that disjunctiveFacetExtraction was in the PR which has since been removed. Fixed 👍🏽